srcpkgs/maturin/template

```diff
@@ -0,0 +1,32 @@
+# Template file for 'maturin'
+pkgname=maturin
+version=0.9.0
+revision=1
+build_style=cargo
+hostmakedepends="python3-setuptools python3-toml"
+depends="python3-toml"
+short_desc="Build and publish crates as python packages"
+maintainer="Andrew J. Hesford <ajh@sideband.org>"
+license="Apache-2.0, MIT"
+homepage="https://github.com/PyO3/maturin"
+distfiles="${homepage}/archive/v${version}.tar.gz"
+checksum=22e8082a743e1dc11f5909b596f9053deb7dc1a56336003677381ba02cf67da8
+
+post_patch() {
+	# setup.py is broken, just use it for the pure python part
+	vsed -e 's/cmdclass.*/packages=["maturin"],/' -i setup.py
+}
+
+post_build() {
+	# python package is pure; the cross environment is not relevant
+	python3 setup.py build
+}
+
+do_check() {
+	: tests fail to build
```

Comment:
Should mention that they fail to build because they use unstable features. That way people might remember to try and re-enable them when we get a new rust version.